metaflow/plugins/aws/awslambda/lambda_cli.py

```diff
@@ -0,0 +1,202 @@
+from logging import Handler
+import click
+import sys
+import os
+
+import boto3
+from botocore.config import Config
+
+from metaflow import R, flowspec
+from metaflow import util
+from metaflow.plugins.aws.batch.batch_cli import sync_metadata_from_s3
+
+from metaflow.exception import (
+    METAFLOW_EXIT_DISALLOW_RETRY,
+)
+
+from .lambda_runner import LambdaRunner, ensure_lambda
+
+from metaflow.metaflow_config import (
+    LAMBDA_LOG_GROUP_NAME,
+    LAMBDA_IMAGE_URI,
+    LAMBDA_ROLE_ARN,
+)
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(help="Commands related to AWS Lambda.")
+def awslambda():
+    pass
+
+
+@awslambda.command(
+    help="Execute a single task using AWS Lambda. This command "
+    "calls the top-level step command inside a AWS Lambda "
+    "job with the given options. Typically you do not "
+    "call this command directly; it is used internally "
+    "by Metaflow."
+)
+@click.argument("step-name")
+@click.argument("code-package-sha")
+@click.argument("code-package-url")
+@click.option("--cpu", type=float, help="CPU requirement for AWS Lambda.")
+@click.option("--memory", type=int, help="Memory requirement for AWS Lambda.")
+@click.option(
```

Comment:
Can we set this option as a global environment variable in `metaflow_config` instead? I presume the value of `throttle_retries` will stay the same across all executions for a user?

Reply:
done in c799b10